torch/testing/_internal/common_methods_invocations.py

```diff
@@ -389,6 +389,25 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_linalg_multi_dot(op_info, device, dtype, requires_grad):
```

Comment:
How long do these tests take? Replacing "S" with 3 would probably decrease the time taken.

Reply:
Overall 10s, I tired replacing S with 3 but there's almost no difference.

Reply:
10s is OK for now.